integration_tests/expr_09.py

```diff
@@ -8,4 +8,27 @@ def main0():
     print(-i1 ^ -i2)
     assert -i1 ^ -i2 == 6
 
+
+def test_multiple_assign_1():
+    a: i32; b:i32; c:i32
```

Comment:
Let's try with variables of different types as well. Something like,
```python
d: f64; e: f32; f: c64; g: i32
c = d = e = g + 1.0
```

Also try with lists, tuples (of the same element types) variables everywhere in the multiple assignment statement.